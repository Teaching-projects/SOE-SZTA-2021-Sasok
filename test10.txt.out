Első egység HP-ja: Első egység DMG-je: Második egység HP-ja: Második egység DMG-je: 
A Csata elkezdődött! Warrior kezdheti a támadást Shaman ellen.
1. körben Warrior támad --- Warrior élete: 98 || Shaman élete: 91
2. körben Shaman támad, ---Warrior élete: 93 || Shaman élete: 91
2. körben Warrior támad --- Warrior élete: 93 || Shaman élete: 85
3. körben Shaman támad, ---Warrior élete: 88 || Shaman élete: 85
3. körben Warrior támad --- Warrior élete: 88 || Shaman élete: 79
4. körben Shaman támad, ---Warrior élete: 83 || Shaman élete: 79
4. körben Warrior támad --- Warrior élete: 83 || Shaman élete: 73
5. körben Shaman támad, ---Warrior élete: 78 || Shaman élete: 73
5. körben Warrior támad --- Warrior élete: 78 || Shaman élete: 67
6. körben Shaman támad, ---Warrior élete: 73 || Shaman élete: 67
6. körben Warrior támad --- Warrior élete: 73 || Shaman élete: 61
7. körben Shaman támad, ---Warrior élete: 68 || Shaman élete: 61
7. körben Warrior támad --- Warrior élete: 68 || Shaman élete: 55
8. körben Shaman támad, ---Warrior élete: 63 || Shaman élete: 55
8. körben Warrior támad --- Warrior élete: 63 || Shaman élete: 49
9. körben Shaman támad, ---Warrior élete: 58 || Shaman élete: 49
9. körben Warrior támad --- Warrior élete: 58 || Shaman élete: 43
10. körben Shaman támad, ---Warrior élete: 53 || Shaman élete: 43
10. körben Warrior támad --- Warrior élete: 53 || Shaman élete: 37
11. körben Shaman támad, ---Warrior élete: 48 || Shaman élete: 37
11. körben Warrior támad --- Warrior élete: 48 || Shaman élete: 31
12. körben Shaman támad, ---Warrior élete: 43 || Shaman élete: 31
12. körben Warrior támad --- Warrior élete: 43 || Shaman élete: 25
13. körben Shaman támad, ---Warrior élete: 38 || Shaman élete: 25
13. körben Warrior támad --- Warrior élete: 38 || Shaman élete: 19
14. körben Shaman támad, ---Warrior élete: 33 || Shaman élete: 19
14. körben Warrior támad --- Warrior élete: 33 || Shaman élete: 13
15. körben Shaman támad, ---Warrior élete: 28 || Shaman élete: 13
15. körben Warrior támad --- Warrior élete: 28 || Shaman élete: 7
16. körben Shaman támad, ---Warrior élete: 23 || Shaman élete: 7
16. körben Warrior támad --- Warrior élete: 23 || Shaman élete: 1
17. körben Shaman támad, ---Warrior élete: 18 || Shaman élete: 1
17. körben Warrior támad --- Warrior élete: 18 || Shaman élete: -5

WARRIOR GYOZEDELMESKEDETT!
