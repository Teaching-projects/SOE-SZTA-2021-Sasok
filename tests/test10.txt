150
3
1.3
140
4
2.3