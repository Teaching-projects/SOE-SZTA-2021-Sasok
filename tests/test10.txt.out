Első egység HP-ja: Első egység DMG-je: Első egység AS-je: Második egység HP-ja: Második egység DMG-je: Második egység AS-je: 1.3
Nem érvényes értéket adott meg, adjon meg egy számot! 
